README.md

```diff
@@ -90,6 +90,9 @@ Here's a list of all the methods available, separated by module:
 * `saveNewMetaproperty(object)`
 * `editMetaproperty(object)`
 * `deleteMetaproperty(object)`
+* `createMetapropertyOption(object)`
```

Comment:
should we change this method to saveNewMetapropertyOption to be consistent with the rest of the methods?

Reply:
Good advice. Changed !